bower.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "purescript-lines",
+  "ignore": [
```

Comment:
This will need a `repository` and valid `license` field for it to be publishable on pursuit. Just steal it from one of the other repos :wink: